Rbac updates (#7126)
DataDog/documentation: content/en/account_management/rbac/_index.md

| **Datadog Standard Role**  | Users are allowed to view and modify all monitoring features that Datadog offers, such as [dashboards][1], [monitors][2], [events][3], and [notebooks][4]. Standard users can also invite other users to organizations.                      |
| **Datadog Read Only Role** | Users do not have access to edit within Datadog. This comes in handy when you'd like to share specific read-only views with a client, or when a member of one business unit needs to share a [dashboard][1] with someone outside their unit. |

## Custom Roles

<div class="alert alert-warning">
Creating and modifying custom roles is an Enterprise feature and is in private beta. <a href="/help">Contact Datadog support</a> to get it enabled for your account.
</div>

Manage your custom roles through the Datadog application, the [Datadog Role API][5], or SAML directly. Find below how to create, update, delete a role. See the [Datadog Role permissions][6] documentation for more information about available permissions.

### Create a custom role

You can create custom roles with:

{{< tabs >}}
{{% tab "Datadog application" %}}

To create a custom role:

> 1. Go to your [Datadog Roles Page][1].

Reviewer: ```suggestion
1. Go to your [Datadog Roles page][1].
```

2. Select **New Role** in the upper right corner of the page.
3. Give a name to your role.

> 4. Optional - Assing a set of permissions to your Role. See the [Datadog Role permissions][2] documentation for more information about available permissions.

Reviewer: ```suggestion
4. Optional - Assigning a set of permissions to your Role. See the [Datadog Role permissions][2] documentation for more information about available permissions.
```

{{< img src="account_management/rbac/create_role.png" alt="Create a custom Role"  style="width:90%;">}}

Once a role is created you can [add this role to existing users][3].

[1]: https://app.datadoghq.com/access/roles
[2]: /account_management/rbac/permissions
[3]: /account_management/users/#edit-a-user-roles
{{% /tab %}}
{{% tab "API" %}}

> Find example to create a Role in the [Datadog Create Role API documentation][1].

Reviewer: ```suggestion
Find an example of how to create a Role in the [Datadog Create Role API documentation][1].
```

[1]: /api/#create-role
{{% /tab %}}
{{< /tabs >}}

### Update a Role

{{< tabs >}}
{{% tab "Datadog application" %}}

To edit a custom role:

> 1. Go to your [Datadog Roles Page][1].

Reviewer: ```suggestion
1. Go to your [Datadog Roles page][1].
```

2. Select the edit button on the Role you would like to modify.
3. Modify the set of permissions to your Role. See the [Datadog Role permissions][2] documentation for more information about available permissions.

> 4. Save your changes

Reviewer: ```suggestion
4. Save your changes.
```

{{< img src="account_management/rbac/edit_role.png" alt="Edit a Role"  style="width:90%;">}}

Once a role is modified, all users who have the role will have their permissions updated.

[1]: https://app.datadoghq.com/access/roles
[2]: /account_management/rbac/permissions
{{% /tab %}}
{{% tab "API" %}}

> Find example to create a Role in the [Datadog Update Role API documentation][1].

Reviewer: ```suggestion
Find an example of how to create a Role in the [Datadog Create Role API documentation][1].
```

[1]: /api/#update-role
{{% /tab %}}
{{< /tabs >}}

### Delete a Role

{{< tabs >}}
{{% tab "Datadog application" %}}

To delete a custom role:

> 1. Go to your [Datadog Roles Page][1].

Reviewer: ```suggestion
1. Go to your [Datadog Roles page][1].
```

2. Select the delete button on the Role you would like to delete.
3. Confirm your decision.

{{< img src="account_management/rbac/delete_role.png" alt="Deleting a Role"  style="width:90%;">}}

{{< img src="account_management/users/delete_role_confirmation.png" alt="Deleting a Role"  style="width:90%;">}}

Once a role is deleted all users who have the role will have their permissions updated. Users without any roles will not be able to use Datadog effectively, but will still maintain limited access. You should always make sure users either have a Role or are disabled if they do not need access to your organization.

[1]: https://app.datadoghq.com/access/roles
{{% /tab %}}
{{% tab "API" %}}

> Find example to create a Role in the [Datadog Delete Role API documentation][1].

Reviewer: ```suggestion
Find an example of how to create a Role in the [Datadog Create Role API documentation][1].
```

</div>

Manage your custom roles through the Datadog application, the [Datadog Role API][5], or SAML directly. Find below how to create, update, delete a role. See the [Datadog Role permissions][6] documentation for more information about available permissions.

### Create a custom role

You can create custom roles with:

{{< tabs >}}
{{% tab "Datadog application" %}}

To create a custom role:

1. Go to your [Datadog Roles Page][1].
2. Select **New Role** in the upper right corner of the page.
3. Give a name to your role.
4. Optional - Assing a set of permissions to your Role. See the [Datadog Role permissions][2] documentation for more information about available permissions.

{{< img src="account_management/rbac/create_role.png" alt="Create a custom Role"  style="width:90%;">}}

Once a role is created you can [add this role to existing users][3].

[1]: https://app.datadoghq.com/access/roles
[2]: /account_management/rbac/permissions
[3]: /account_management/users/#edit-a-user-roles
{{% /tab %}}
{{% tab "API" %}}

Find example to create a Role in the [Datadog Create Role API documentation][1].

[1]: /api/#create-role
{{% /tab %}}
{{% tab "SAML" %}}

> Custom roles are imported via SAML integrations from identity providers. Datadog receives the user groups from your IdP and automatically generates roles to match those user groups. In this case, users who sign in via the IdP are automatically associated with those roles and have the permissions that are granted to those roles.

Reviewer 2: This functionality is being deprecated. Roles must be explicitly created either via the API or the Application going forward.

Reviewer 3: Removed that section
